nil/services/rpc/jsonrpc/send_transaction.go

```diff
@@ -19,12 +22,33 @@ func (api *APIImpl) SendRawTransaction(ctx context.Context, encoded hexutil.Byte
 
 	shardId := extTxn.To.ShardId()
 	reason, err := api.rawapi.SendTransaction(ctx, shardId, encoded)
+
+	headers, ok := ctx.Value(transport.HeadersContextKey).(http.Header)
+	if !ok {
+		headers = http.Header{}
+		headers.Add("Client-Type", "failed to extract headers from context")
+	}
+
+	// even though warn here looks weird these logs shouldn't be
+	// occasionally gone when global log level is increased
+	log := api.clientEventsLog.Warn().
```

Comment:
Why `warn`?

Reply:
Trace/Debug/Info

Reply:
I left the comment describing why it's preferable to keep warn level here